build.gradle.kts

```diff
@@ -29,10 +27,15 @@ plugins {
     id("org.jetbrains.dokka") apply false
 
     id("com.github.ben-manes.versions")
-    id("org.ajoberstar.grgit")
+    id("org.ajoberstar.reckon")
     id("org.jetbrains.gradle.plugin.idea-ext")
 }
 
+reckon {
+    scopeFromProp()
+    stageFromProp("beta", "rc", "final")
```

Comment:
Please add a link to the plugins documentation to the commit message, this would help understanding the configuration.

Reply:
Done.